java/src/jmri/jmrit/symbolicprog/NameFile.java

```diff
@@ -45,18 +45,6 @@ public Element elementFromName(String name) {
         return _nameHash.get(name);
     }
 
-    /**
-     *
-     * @return the default instance of this class
-     * @deprecated since 4.9.2; use
-     * {@link jmri.InstanceManager#getDefault(java.lang.Class)} instead
-     */
-    @Deprecated
-    public synchronized static NameFile instance() {
```

Comment:
The header of the class has the text:
```
 * Initially, we only need one of these, so we use an "instance" method to
 * locate the one associated with the "xml/names.xml" file.
```
Maybe that text should be updated as well, since the method `instance()` is removed?